osu.Desktop/DiscordRichPresence.cs

```diff
@@ -121,6 +151,41 @@ private void updateStatus()
                 presence.Details = string.Empty;
             }
 
+            if (!hideIdentifiableInformation && multiplayerClient.Room != null)
+            {
+                MultiplayerRoom room = multiplayerClient.Room;
+
+                if (room.Users.Count == usersCurrentlyInLobby)
+                    return;
```

Comment:
This is incorrect. If the activity has changed or the `LargeImageText` property below this block of code has changed then that will not be reflected until the presence is updated. `updateStatus` needs to be refactored, I will take a look into that to speed things up at this point.